tests/config_types/test.bats

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
This repo is Apache licensed.

Reply:
Aw - good catch! Will sort out all the license headers.